google/datalab/bigquery/commands/_bigquery.py

```diff
@@ -155,6 +155,7 @@ def _create_query_subparser(parser):
   query_parser.add_argument('--datasources', help='List of external datasources to reference in the query body',
                             nargs='+')
   query_parser.add_argument('--subqueries', help='List of subqueries to reference in the query body', nargs='+')
+  query_parser.add_argument('--parameters', help='List of subqueries to reference in the query body', nargs='+')
```

Comment:
I think parameters realistically need to go into the cell body.

1. They won't necessarily fit on the one line.
2. You need to support both positional as well as named parameters. Commandline won't help with latter.